### PR TITLE
Updated " C++/fah_to_cel_function(CB).cpp " 

### LINKED ISSUE #622: C++/fah_to_cel_function(CB).cpp [Required output not received]
The datatype of c (Celcius) in C++/fah_to_cel_function(CB).cpp has int datatype. So the output is rounded to the lower integer which for most of the cases is giving incorrect output.

### PR DESCRIPTION
**Pull request by arpitbaheti2002**
Hi, 
I went through the C++ program that converts temperature from Fahrenheit scale to Celcius scale (C++/fah_to_cel_function(CB).cpp) and found an issue in it.

The function fahToCel() has an integer variable c that stores temperature in Celcius. Now, if I convert 100 deg. Fahrenheit to Celcius the output should be 37.78 but due to integer datatype, it is displaying 37.

Fixes: #622

### CODE REVIEW

**Comment by arpitbaheti2002** [comment]
I have updated my description. Can you please review it?

File: C++/fah_to_cel_function(CB).cpp [hunk 1]
```diff
@@ -1,9 +1,10 @@
 #include<iostream>
+#include<iomanip>
 using namespace std;
 void fahToCel(int f,int steps){
     for(;f<=300;f+=steps){
-        int c = 5*(f-32)/9;
-        cout<<f<<" "<<c<<endl;
+        float c = 5*(f-32)/9.0;
+        cout<<f<<" "<<fixed<<setprecision(2)<<c<<endl;
```
Comment (kaustubhgupta) [inline C++/fah_to_cel_function(CB).cpp]: Add the line, "Temperature in Celcius: " in print statement before the output value

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: C++/fah_to_cel_function(CB).cpp (hunk 1 above)
Comment (arpitbaheti2002) [reply C++/fah_to_cel_function(CB).cpp]: I have successfully updated the print statement. Thank you for the feedback.

**Review by kaustubhgupta — APPROVED** [review]
Good to go!

**Pull request merged**